.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,31 +1,40 @@
-# Description of the problem
+# Description of the issue
 
-- Explain the previous issue (description of the issue or the reference number of an open issue)
-- Explain the need for a new feature or component
-- Describe the added code
+- Describe the previous issue
+- Explain the need for a new feature, component or example
 
 # Description of the solution
 
 - Explain the solution to the existing issue
 - Explain the content of the new feature
 
-# How Has This Been Tested?
+# How has this been tested?
 
-- List and describe the test(s) for the added code (if there is no test for the pull request, explain why (i.e. the test already exists but was changed))
-
-- [ ] Test A
-- [ ] Test B
+- List and describe the new test(s) (i.e. what feature/parameter does it test)
+- Explain the changes and/or impacts on current tests
+- If not related to tests, how did you ensure that the solution works?
```

Comment:
```suggestion
- If not related to tests, how did you ensure that the solution works and how will you ensure that it will continue to work in the future?
```

Reply:
I understand why it is good to know, but I think this is something we can hardly anwser. I feel nobody will keep that one. Do you have any example?

Reply:
Yeah you are right. It's wishful thinking more than anything else. Maybe this is not necessary in the end.